### PR TITLE
Update examples/maven/jdt-ecj-settings/README.md

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

**Comment by vorburger** [comment]
@sylvainlaurent I just noticed https://github.com/lastnpe/eclipse-null-eea-augments/pull/34/commits/dbd13aa0ca7802e3cbb0f6c351ae0434d4c7fc17 and thought it was useful to clarify this..

File: examples/maven/jdt-ecj-settings/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 The `org.eclipse.jdt.core.prefs` file in this folder enables stricter checks of java code, especially in the area of null analysis.
 
-It should be placed in the `.settings` directory of each eclipse project for which these checks should be applied.
+It should either be manually placed in the `.settings` directory of each eclipse project for which these checks should be applied,
+or you can use the https://github.com/lastnpe/eclipse-external-annotations-m2e-plugin which can do this automatically for you,
+if appropriately configured to do so (the example projects here are).
```
Comment (kwin) [inline examples/maven/jdt-ecj-settings/README.md]: the example projects here are -> the example project are here
A link would be helpful as well.

Comment (vorburger) [reply examples/maven/jdt-ecj-settings/README.md]: @kwin no, not "are here", but "the example projects here are so configured [with this]" .. ;-) but slightly edited now, hopefully better? And thanks for reviewing!

Comment (kwin) [reply examples/maven/jdt-ecj-settings/README.md]: Ah, ok, got it.

**Comment by sylvainlaurent** [comment]
Your PR just made me discover this feature of the eclipse-external-annotations-m2e-plugin which I totally ignored. Though the feature may be convenient for eclipse as IDE, it cannot be used when invoking compilation from the command-line. AFAIK you need the `org.eclipse.jdt.core.prefs`to be present in `.settings`. It might be nice to clarify this even more.

**Comment by vorburger** [comment]
> feature of the eclipse-external-annotations-m2e-plugin which I totally ignored

@sylvainlaurent well this feature IS [clearly documented on the README](https://github.com/lastnpe/eclipse-external-annotations-m2e-plugin/blob/master/README.md): 

>> Allows to configure Java Compiler Project Properties from maven-compiler-plugin; read either from a dependency of maven-compiler-plugin containing a org.eclipse.jdt.core.prefs file, or from configuration/compilerArguments/properties.

> cannot be used when invoking compilation from the command-line

Yeah, you actually totally can... ;-) Thanks to the MAGIC with the maven-dependency-plugin that puts it into `target/` and  <compilerArguments><properties>${project.build.directory}/org.eclipse.jdt.core.prefs as done in the [examples pom.xml](https://github.com/lastnpe/eclipse-null-eea-augments/blob/master/examples/maven/parent/pom.xml) .. IMHO the way these things typically work is that you can document all you want (case in point here?) and anyone who's really interested how this was set up peeks into the pom.xml sources anyway? :smile_cat: 

Therefore I think this is sufficient as is - merge?

**Pull request merged**